ENVIRONMENT.md

```
@@ -0,0 +1,163 @@
+uv pip install "torch<2.4" "torchaudio<2.4" --index-url https://download.pytorch.org/whl/cu118
+
+# 4. その他の依存関係
+uv pip install -r requirements.txt
+
+# 5. 初期化
+python initialize.py
+```
+
+### Linux/macOS
+```bash
+# 既存の仮想環境がある場合は削除
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)